python/tk_multi_loader/ui/dialog.py

```diff
@@ -182,18 +183,33 @@ def setupUi(self, Dialog):
         self.thumbnail_mode.setObjectName("thumbnail_mode")
         self.horizontalLayout_2.addWidget(self.thumbnail_mode)
         self.list_mode = QtGui.QToolButton(Dialog)
+        self.list_mode.setMinimumSize(QtCore.QSize(0, 26))
         icon2 = QtGui.QIcon()
         icon2.addPixmap(QtGui.QPixmap(":/res/mode_switch_card.png"), QtGui.QIcon.Normal, QtGui.QIcon.Off)
         self.list_mode.setIcon(icon2)
         self.list_mode.setCheckable(True)
         self.list_mode.setObjectName("list_mode")
         self.horizontalLayout_2.addWidget(self.list_mode)
+        self.label_5 = QtGui.QLabel(Dialog)
+        self.label_5.setMinimumSize(QtCore.QSize(5, 0))
+        self.label_5.setMaximumSize(QtCore.QSize(5, 16777215))
+        self.label_5.setText("")
+        self.label_5.setObjectName("label_5")
+        self.horizontalLayout_2.addWidget(self.label_5)
+        self.search_publishes = QtGui.QToolButton(Dialog)
```

Comment:
It would be nice if hitting command-F on Mac and ctrl-F on other OSes if it showed the search box.